src/Fantomas.Tests/ModuleTests.fs

```diff
@@ -437,4 +437,21 @@ type UrlModel =
     { IsFsi: bool
       KeepNewlineAfter: bool
       Defines: string }
-"""
\ No newline at end of file
+"""
+
+[<TestCase(true)>]
```

Comment:
Nitpick: I'd prefer a not using the `TestCase` attribute but duplicate the test to SignatureTests.fs.
To remain consistent with the rest of the codebase and to easier spot regressions.
This is clear now but it will be confusing that a test fails in ModuleTests while changing something in Signature AST for example.

Reply:
Will do!